npm/react/examples/nextjs/next.config.js

```diff
@@ -1,4 +1,3 @@
 const withMDX = require('@next/mdx')()
-const withSass = require('@zeit/next-sass')
 
-module.exports = withSass(withMDX())
+module.exports = withMDX()
```

Comment:
I removed this, because Next.js will only use the built-in CSS modules support if you do not provide your own CSS preprocessing pipeline. This lets us test that the change I made (custom `index-template.html`) is working.